float32 time_to_catch
string game_status
